lib/vsc/mympirun/mpi/mpi.py

```diff
@@ -572,8 +574,21 @@ def make_node_file(self):
             self.log.raiseException(msg)
 
     def get_universe_ncpus(self):
```

Comment:
@Caylo please add tests for this method, should be fairly easy to test in isolation?